utilities/rocAL/rocAL_unittests/testAllScripts.sh

```diff
@@ -0,0 +1,163 @@
+#!/bin/bash
+cwd=$(pwd)
+if [ -d build ];then 
+    rm -rf ./build/*
+else
+    mkdir build
+fi
+cd build
+cmake ..
+make -j$nproc
+
+if [$ROCAL_DATA_PATH == ""]
+then 
+    echo "Need to export ROCAL_DATA_PATH"
+    exit
+fi
+
+image_path=${ROCAL_DATA_PATH}/rocal_data/coco/coco_10_img/train_10images_2017/
+coco_classification_path=${ROCAL_DATA_PATH}/rocal_data/coco/coco_10_img/train_10images_2017/
+coco_detection_path=${ROCAL_DATA_PATH}/rocal_data/coco/coco_10_img/train_10images_2017/
+tf_classification_path=${ROCAL_DATA_PATH}/rocal_data/tf/classification/
+tf_detection_path=${ROCAL_DATA_PATH}/rocal_data/tf/detection/
+caffe_path=${ROCAL_DATA_PATH}/rocal_data/caffe/classification/
```

Comment:
@fiona-gladwin What about detection? it is using the same folder in lines 99-101 and lines 104-107 below. Please check.